PRODUCTIONISATION.md

```diff
@@ -19,26 +19,30 @@
 
 Ensuring that your project adheres to the following RFCs is a good first step as these will be referenced further below:
 
+* [RFC 008 - Server naming convention][rfc_008]
 * [RFC 011 - Editorconfig][rfc_011]
 * [RFC 012 - Makefile standards][rfc_012]
 * [RFC 013 - Code repository standards][rfc_013]
 
 ### 1. General requirements
 
 * [ ] 1.1 Application source code hosted with a Git VCS service
-* 1.2 No linting violations
-  * [ ] 1.2.1 No style related linting violations and no overrides permitted
-  * [ ] 1.2.2 No complexity related linting violations. Overrides permitted but must be commented inline with the code it is overriding
+* 1.2 Adhere to project coding standards
+  * [ ] 1.2.1 Provides a style guideline
+  * [ ] 1.2.2 Consistency within development team
+  * [ ] 1.2.3 If applicable, no linting violations (style and complexity related)
 * [ ] 1.3 `.editorconfig` as per [RFC 011][rfc_011]
 * [ ] 1.4 `Makefile` as per [RFC 012][rfc_012]
 * [ ] 1.5 `README.md` as per [RFC 013][rfc_013]
 * [ ] 1.6 `README.md` to also include references for all 3rd party service integrations
-* 1.7 Local development seeds from production:
-  * [ ] 1.7.1 Seed a specific item (for example a product, in an ecom system)
+* 1.7 Local development seeds from production-like data (similar sized datasets - to assist with debugging and for diagnosing unusual performance regressions):
+  * [ ] 1.7.1 Seed a specific item (for example, a product in an ecom system)
   * [ ] 1.7.2 Seed a specified number of items in batch
   * [ ] 1.7.3 Seed full catalogue of items
-* [ ] 1.8 Local development seed fixtures (for example, using ffaker for Ruby/Rails)
+* [ ] 1.8 Local development seed fixtures (for example, using Faker or AutoFixture)
 * [ ] 1.9 Install an application profiler for the `development` environment so that metrics such as page render times, database query times can be used to make performance improvements
+* [ ] 1.10 Infrastructure to implement auto-scaling for production where possible, otherwise refer to the [Infrastructure monitoring](#8-infrastructure-monitoring) section
```

Comment:
I think we should lose the "where possible". in here.

we shouldn't be in a situation where auto-scaling isn't possible, as this implies manual scaling isn't possible either (usually due to poorly architectured software).

It may be worth moving `1.10` into `8.x`, as they're closely related. May need `8` renaming though.
Thoughts, @davidwinter ?

Reply:
I was considering `8.x` at first but I think this is better as a general requirement(?).

Reply:
Also removed "where possible" in https://github.com/madetech/productionisation/pull/51/commits/b668aa12022d7effb29ddca3c9587dae8d4b0ef7.

Reply:
The problem with "General requirements" is it amounts to a grouping of unrelated statements.

We should aim to logically group everything, and treat "General" as an emergency situation of "this doesn't fit anywhere", otherwise it makes the section to read as a whole (lots of context switching in one place)

Reply:
That makes sense. Any suggestions on what to rename `8` tho?

Reply:
`Infrastructure Monitoring + Scaling`?
they do go hand in hand

Reply:
See https://github.com/madetech/productionisation/pull/51/commits/adc69223fd081848e144c138dd4b1cc456d26e89 for changes.